grassroots-shared/src/decorators/Trim.decorator.ts

```diff
@@ -0,0 +1,19 @@
+import { Transform } from "class-transformer";
```

Comment:
I want participant display names to be unique without worrying about whitespace.